charts/lightdash/README.md

```diff
@@ -54,7 +54,7 @@ If you don't want helm to manage this, you may wish to separately create a secre
 | backendConfig.create | bool | `false` |  |
 | browserless-chrome.enabled | bool | `true` |  |
 | browserless-chrome.env.CONNECTION_TIMEOUT | string | `"180000"` |  |
-| browserless-chrome.image.tag | string | `""` |  |
+| browserless-chrome.image.tag | string | `"1.61.1-chrome-stable"` |  |
```

Comment:
I'm surprised this `latest` version is 1 year old... 

anyhow, I tested this version  locally and it is working as expected